Init Go module and add assets package

jrick opened the pull request:
The assets package may be used to get the absolute filepath of any
file or directory relative root of the module.  This will be used by
the release builder to access the site contents for creating dexc
distributions.

chappjc commented on go.mod:
```diff
@@ -0,0 +1,3 @@
+module decred.org/dcrdex-assets
```
@dajohi 

```
$  curl https://decred.org/dcrdex-assets?go-get=1
<meta name="go-import" content="decred.org/dcrdex git https://github.com/decred/dcrdex">
```

Can we revise this for the dcrdex-assets repo?  This needs adjusting, correct, @jrick?

jrick replied:
yes, it's matching incorrectly. will need some fixes on the webserver, but nothing i can do here on this pr about it.

chappjc replied:
Understood, was just poking around

chappjc reviewed (approved):
OK, but does https://github.com/decred/dcrweb/pull/928 need to actually be deployed before this go.mod hits master in this repo?

jrick commented:
it already is

chappjc commented:
Wow, I tried 4 seconds ago and it wasn't.

jrick committed:
Init Go module and add assets package

The assets package may be used to get the absolute filepath of any
file or directory relative to the root of the module.  This will be
used by the release builder to access the site contents for creating
dexc distributions.
